api/api.go

```diff
@@ -461,6 +523,159 @@ func CreateOrb(ctx context.Context, logger *logger.Logger, namespace string, nam
 	return orb, err
 }
 
+func incOrbVersion(orbVersion string, segment string) (string, error) {
```

Comment:
This function is not really related to the API.  Maybe add a TODO to move it to some other package?